src/components/Cardgrid.tsx

```diff
@@ -0,0 +1,81 @@
+import React from 'react';
+import {
+  CircularProgress,
+  FontIcon,
+  IconButton,
+} from 'material-ui';
+import {
+  GridList,
+  GridTile
+} from 'material-ui/GridList';
+import { muiThemeable } from 'material-ui/styles';
+
+const styles = {
+  image: {
+    width: 'initial',
+    height: 'initial',
+    maxWidth: '100%',
+    maxHeight: '100%',
+  },
+  load: {
+    width: '100%',
+    height: 135,
+    paddingTop: 40,
+    boxSizing: 'border-box',
+    backgroundColor: '#606060',
+    textAlign: 'center',
+  },
+  loadmore: {
+    color: 'white',
+    fontSize: 30,
+    cursor: 'pointer',
+    textDecoration: 'underline',
+    backgroundColor: 'transparent',
+    border: 'none',
+    outline: 'none',
+  }
+};
+
+export default muiThemeable()(Cardgrid);
+
+function Cardgrid({
+  data,
+  loadMore,
+  loading,
+  muiTheme,
+}) {
```

Comment:
Props for me (for inline styling):

- it's consistent with `material-ui`
- I'm focused only on `javascript` no worry about css, sass, precompiles, etc.
- All components code is in one place
- I'm used to this approach because I worked with React Native ;)